From 862e805067c5e3a7162a0930af03fe3fb7249d38 Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Mon, 7 Aug 2017 01:00:15 -0500
Subject: [PATCH 2321/5242] MLK-16149 arm64: dts: imx8qm/qxp: Remove memory
 reserve entry for M4 and ATF

commit  b064514a779371f48a7ea3b681d9804130e3146e from
https://source.codeaurora.org/external/imx/linux-imx.git

Since u-boot will process the owned memory in current partition and modify
the DTB memory nodes for kernel at runtime, no need to set the first 128MB memory
as "memreserve" in DTS.

ATF: 0x80000000 -- 0x801fffff.
 - This area will be cleaned out of available memory by u-boot.

M4:  0x88000000 -- 0x8fffffff.
 - A memory reserve entry will be added to DTB by u-boot.

Signed-off-by: Ye Li <ye.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |    4 ----
 .../boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts |    4 ----
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |    4 ----
 3 files changed, 12 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 7e0ae18..0f23564 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -15,10 +15,6 @@
 
 /dts-v1/;
 
-/* First 128KB is for PSCI ATF. */
-/* Last 127M is for M4/RPMSG */
-/memreserve/ 0x80000000 0x08000000;
-
 #include "fsl-imx8qm.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
index a688333..ced921e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-lpddr4-arm2.dts
@@ -14,10 +14,6 @@
 
 /dts-v1/;
 
-/* First 128KB is for PSCI ATF. */
-/* Last 127M is for M4/RPMSG */
-/memreserve/ 0x80000000 0x08000000;
-
 #include "fsl-imx8qxp.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 94e9990..bb79c9428 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -14,10 +14,6 @@
 
 /dts-v1/;
 
-/* First 128KB is for PSCI ATF. */
-/* Last 127M is for M4/RPMSG */
-/memreserve/ 0x80000000 0x08000000;
-
 #include "fsl-imx8qxp.dtsi"
 
 / {
-- 
1.7.9.5

